[📚 DOCS]: Fix for English Wording, Grammar, Spelling, and Clarity (#1805)
AppFlowy-IO/AppFlowy: README.md

<p align="center"><img src="https://github.com/AppFlowy-IO/appflowy/blob/main/doc/imgs/howtostar.gif" alt="AppFlowy Github - how to star the repo" width="100%" /></p>

## Getting Started with development

Please view the [documentation](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy) for OS specific development instructions

## Roadmap

* [AppFlowy Roadmap ReadMe](https://appflowy.gitbook.io/docs/essential-documentation/roadmap)

> * [AppFlowy Public Roadmap](https://github.com/orgs/AppFlowy-IO/projects/5/views/12)

Reviewer: These two are identical in practice, but what's the reason behind this change?

Author: It's a markdown linting convention. 

The convention for unordered list:

```
* Item 1
  + Item 2
    - Item 3
  + Item 4
* Item 4
  + Item 5
```

[See Markdownlint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md004---unordered-list-style)

Author: @richardshiue what do you think? should we follow this linting or just do as we like? I didn't see any markdownlint config in this repo, so I think, it should be declared for consistency.

Reviewer: Interesting, I didn't know this existed. I personally don't have a preference so let's see what the maintainers say. Perhaps we can add the Github Action fort this as well. Apart from this, LGTM.

If your Pull Request is accepted as it fixes a bug, adds functionality, or makes AppFlowy's codebase significantly easier to use or understand, **Congratulations!** If your administrative and managerial work behind the scenes sustains the community, **Congratulations!** You are now an official contributor to AppFlowy. Get in touch with us ([link](https://tally.so/r/mKP5z3)) to receive the very special Contributor T-shirt!
Proudly wear your T-shirt and show it to us by tagging [@appflowy](https://twitter.com/appflowy) on Twitter.

## Join the community to build AppFlowy together

<a href="https://github.com/AppFlowy-IO/AppFlowy/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=AppFlowy-IO/AppFlowy" />
</a>

## Why Are We Building This?

Notion has been our favourite project and knowledge management tool in recent years because of its aesthetic appeal and functionality. Our team uses it daily, and we are on its paid plan. However, as we all know, Notion has its limitations. These include weak data security and poor compatibility with mobile devices. Likewise, alternative collaborative workplace management tools also have their constraints.

> The limitations we encountered using these tools rooted in our past work experience with collaborative productivity tools led to our firm belief that there is, and will be, a glass ceiling on what's possible in the future for tools like Notion. This emanates from these tools probable struggles to scale horizontally at some point. They will likely be forced to prioritize a proportion of customers whose needs differ from the rest. While decision-makers want a workplace OS, the truth is that it is impossible to come up with a one-size fits all solution in such a fragmented market.

Reviewer: ```suggestion
The limitations we encountered using these tools and our past work experience with collaborative productivity tools have led to our firm belief that there is a glass ceiling on what's possible for these tools in the future. This emanates from the fact that these tools will probably struggle to scale horizontally at some point and be forced to prioritize a proportion of customers whose needs differ from the rest. While decision-makers want a workplace OS, it is impossible to come up with a one-size fits all solution in such a fragmented market.
```

This paragraph sounds off and there are some parts I don't completely understand. It's just a suggestion, so do with it what you will.

Author: I love ❤️ this suggestion, thanks.

If you'd like to propose a feature, submit a feature request [here](https://github.com/AppFlowy-IO/AppFlowy/issues/new?assignees=&labels=&template=feature_request.yaml&title=%5BFR%5D+) <br/>
If you'd like to report a bug, submit a bug report [here](https://github.com/AppFlowy-IO/AppFlowy/issues/new?assignees=&labels=&template=bug_report.yaml&title=%5BBug%5D+)

## **Releases**

Please see the [changelog](https://www.appflowy.io/whatsnew) for more details about a given release.

## Contributing

Contributions make the open-source community a fantastic place to learn, inspire, and create. Any contributions you make are **greatly appreciated**. Please look at [Contributing to AppFlowy](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/contributing-to-appflowy) for details.

> If your Pull Request is accepted as it fixes a bug, adds functionality, or makes AppFlowy's codebase significantly easier to use or understand. Congratulations! If your administrative and managerial work behind the scenes sustains the community. Congratulations! You are now an official contributor to AppFlowy. Get in touch with us ([link](https://tally.so/r/mKP5z3)) to receive the very special Contributor T-shirt!

Reviewer: Could this be rephrased? This becomes a dangling dependent clause.

Author: It solved now
